src/gesture-recognizers.js

```diff
@@ -280,6 +280,10 @@ class SwipeRecognizer extends GestureRecognizer {
   /** @override */
   onTouchStart(e) {
     const {touches} = e;
+    // If already eventing, ignore additional touches
+    if (this.eventing_ && touches && touches.length > 1) {
+      return true;
+    }
     if (touches && touches.length == 1) {
```

Comment:
There are two options here: 
1. Early return for additional touch case and continue listening to gesture. Slightly suboptimal UX because additional touch will result in gesture stopping until you lift the finger. 

```
    // If already eventing, ignore additional touches
    if (this.eventing_ && touches && touches.length > 1) {
      return true;
    }
```

2. Actually ignore the touch. 

```
if (touches && touches.length >= 1) 
```
This is better UX for a single gesture, but may result in confusion between swipe and pinch.

Reply:
Confusion between swipe and pinch context here: https://github.com/ampproject/amphtml/pull/12219/files#r153873499. But I think this was already mitigated by adding a threshold to not trigger pinch if both fingers move in the same direction in `PinchRecognizer`'s `onTouchMove` function.